Dockerfile

```diff
@@ -25,27 +25,28 @@ RUN pip install -r requirements/production.txt
 
 
 # Stage 2 - Install frontend deps and build assets
-FROM mhart/alpine-node:10 AS frontend-build
+FROM mhart/alpine-node:12 AS frontend-build
```

Comment:
Node 12 is already EOL I believe, so it might be better to upgrade to 16+ (https://nodejs.org/en/about/releases/), but this is probably something for a future PR